standalone/node/src/cli.rs

```diff
@@ -16,38 +16,34 @@
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
-use sc_cli::{KeySubcommand, RunCmd, SignCmd, VanityCmd, VerifyCmd};
-use structopt::StructOpt;
-
 /// An overarching CLI command definition.
-#[derive(Debug, StructOpt)]
+#[derive(Debug, clap::Parser)]
 pub struct Cli {
 	/// Possible subcommand with parameters.
-	#[structopt(subcommand)]
+	#[clap(subcommand)]
 	pub subcommand: Option<Subcommand>,
+
 	#[allow(missing_docs)]
-	#[structopt(flatten)]
-	pub run: RunCmd,
-	#[allow(missing_docs)]
-	#[structopt(long, help = "Custom block duration in milliseconds (only useful with --dev)")]
+	#[clap(flatten)]
+	pub run: sc_cli::RunCmd,
+
+	/// Custom block duration in milliseconds (only useful with --dev)
```

Comment:
Will it show the help message in the cli or just docstring?

Reply:
```
--block-millisecs <BLOCK_MILLISECS>
            Custom block duration in milliseconds (only useful with --dev)
```

yes!